code/game/objects/structures/electricchair.dm

```diff
@@ -48,10 +48,10 @@
 	s.set_up(12, 1, src)
 	s.start()
 	if(buckled_mob)
-		buckled_mob.burn_skin(85)
+		buckled_mob.electrocute_act(85, src, 1)

```

Comment:
Well, technically you don't need the `1` argument either, since siemens_coefficient defaults to `1`.